.github/workflows/e2e-tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: make -f x.mk e2e-local NODES=2 JUNIT_DIRECTORY=./artifacts/
+      - run: make -f Makefile e2e-local NODES=2 JUNIT_DIRECTORY=./artifacts/
```

Comment:
You can just remove `-f Makefile` entirely

Reply:
happy to remove it... my rationale for leaving it was to make it explicit

Reply:
fixed